client/packages/inventory/src/Stocktake/ListView/CreateStocktakeButton.tsx

```diff
@@ -49,6 +49,27 @@ export const CreateStocktakeButton: React.FC<{
   const [createStocktakeArgs, setCreateStocktakeArgs] =
     useState<CreateStocktakeArgs>(DEFAULT_ARGS);
 
+  const generateComment = () => {
```

Comment:
Kind of feel like this is back end task (master list name or location name is straight away available). Also you can add items after stocktake was created, would this confuse people (having comment that it was created form certain master list or location and but having items that are not in the master list or location) ?

Sorry not asking for a change, but since it's a new addition, wanted to review it

Reply:
I'm on the fence - was thinking it should be a b/e task and now rather like having the comment template in the f/e because we can translate it.

and I think its ok that the comment says 'created from master list x'  and then you have some items added later. Ideally the log would show that items have been added.. but that can come later! ( it doesn't currently add log entries for changes like that )